src/utils/crud/channel.js

```diff
@@ -0,0 +1,43 @@
+const createChannel = async ({database, channel, user}) => {
+	const {name, slug} = channel
+	const {id: user_id} = user
+	const now = new Date()
+
+	return await database
```

Comment:
except inside try/catch, there's no need to `return await`. AFAIK should be

```suggestion
	return database
```

See https://jakearchibald.com/2017/await-vs-return-vs-return-await/

Reply:
nice, thx for th elink